Form/TypeExtension/FileUploadFormExtension.php

```diff
@@ -31,41 +31,40 @@ public function __construct(FSiUploadableRequestHandler $requestHandler)
         $this->requestHandler = $requestHandler;
     }
 
-    /**
-     * {@inheritdoc}
-     */
+    public static function getExtendedTypes()
+    {
+        return [self::formClass()];
```

Comment:
this method does not exist in symfony < 4.2, so it could return `FormType::class` safely